crates/ethereum/engine-primitives/src/payload.rs

```diff
@@ -28,7 +29,7 @@ pub struct EthBuiltPayload {
     pub(crate) fees: U256,
     /// The blobs, proofs, and commitments in the block. If the block is pre-cancun, this will be
     /// empty.
-    pub(crate) sidecars: Vec<BlobTransactionSidecar>,
+    pub(crate) sidecars: Vec<BlobTransactionSidecarVariant>,
```

Comment:
hmm, storing as vec of variant isn't super ideal because we only ever have a vec of a single variant, so I think we could use an `enum BlobSidecars{vec,vec}` instead?

Reply:
sure